specification/migrate/resource-manager/Microsoft.Migrate/2017-11-11-preview/migrate.json

```diff
@@ -0,0 +1,2940 @@
+          "type": "string",
+          "description": "Expiry date of download url.",
+          "readOnly": true
+        }
+      }
+    },
+    "ProjectResultList": {
+      "type": "object",
+      "description": "List of projects.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of projects.",
+          "items": {
+            "$ref": "#/definitions/Project"
+          }
+        }
+      }
+    },
+    "MachineResultList": {
+      "type": "object",
+      "description": "List of machines.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of machines.",
+          "items": {
+            "$ref": "#/definitions/Machine"
+          }
+        }
+      }
+    },
+    "GroupResultList": {
+      "type": "object",
+      "description": "List of groups.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of groups.",
+          "items": {
+            "$ref": "#/definitions/Group"
+          }
+        }
+      }
+    },
+    "AssessedMachineResultList": {
+      "type": "object",
+      "description": "List of assessed machines.",
+      "properties": {
+        "value": {
+          "description": "List of assessed machines.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AssessedMachine"
+          }
+        }
+      }
+    },
+    "AssessmentResultList": {
+      "type": "object",
+      "description": "List of assessments.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of assessments.",
+          "items": {
+            "$ref": "#/definitions/Assessment"
+          }
+        }
+      }
+    },
+    "OperationResultList": {
+      "type": "object",
+      "description": "List of API operations.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of operations.",
+          "items": {
+            "$ref": "#/definitions/Operation"
+          }
+        }
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.Migrate/projects": {
+      "get": {
+        "security": [
+          {
+            "azure_auth": [
+              "user_impersonation"
+            ]
+          }
+        ],
+        "summary": "Get all projects.",
+        "description": "Get all the projects in the resource group.",
+        "operationId": "Projects_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/acceptLanguage"
+          }
+        ],
+        "produces": [
+          "application/json"
+        ],
```

Comment:
Since this was already added globally, it is not needed to override

Reply:
Same here, this applies to all operations on this swagger. If the consumes/produces was already defined globally with the same value, it is not needed to override it on every operation.

---
In reply to: [167111806](https://github.com/Azure/azure-rest-api-specs/pull/2446#discussion_r167111806) [](ancestors = 167111806)

Reply:
Done.

Reply:
Fixed